tests/test_locking.py

```diff
@@ -0,0 +1,107 @@
+import uuid
+
+import pytest
+
+from django.db import connection, models, transaction
+
+from psqlextra.locking import (
+    PostgresTableLockMode,
+    postgres_lock_model,
+    postgres_lock_table,
+)
+
+from .fake_model import get_fake_model
+
+
+@pytest.fixture
+def mocked_model():
+    return get_fake_model(
+        {
+            "name": models.TextField(),
+        }
+    )
+
+
+def get_table_locks():
+    with connection.cursor() as cursor:
+        return connection.introspection.get_table_locks(cursor)
+
+
+@pytest.mark.django_db(transaction=True)
+def test_postgres_lock_table(mocked_model):
+    lock_signature = (
+        "public",
+        mocked_model._meta.db_table,
+        "AccessExclusiveLock",
```

Comment:
What using `@pytest.mark.parametrize` with `PostgresTableLockMode` to ensure that all lock modes are working?